lib/vagrant-openshift/command/test_openshift3_image.rb

```diff
@@ -103,31 +103,20 @@ def execute
 # switch to the desired ref
 git checkout #{ref}
 
-# If software version is set, change directory to it
-[ -n "#{image_version}" ] && pushd #{image_version} > /dev/null
+if [ #{image} != "sti-base" ]; then
+  # Pull base images
+  docker pull ci.dev.openshift.redhat.com:5000/openshift/base-centos7 && docker tag ci.dev.openshift.redhat.com:5000/openshift/base-centos7 openshift/base-centos7
+  docker pull ci.dev.openshift.redhat.com:5000/openshift/base-rhel7 && docker tag ci.dev.openshift.redhat.com:5000/openshift/base-rhel7 openshift/base-rhel7
+fi
 
-# Build the STI image we use for testing
-docker build -t #{image}-candidate .
+# Test CentOS 7 based images with STI test framework
+make test VERSION=#{image_version}
```

Comment:
we said that the only convention would be to have 'test/run', having this in place changes that convention to have 'make test VERSION=??'  command...